src/web/routes/application/enter-code/validate.js

```diff
@@ -1,9 +1,11 @@
+const { CONFIRMATION_CODE_SESSION_PROPERTY } = require('../common/constants')
 const { check } = require('express-validator')
 const { translateValidationMessage } = require('../common/translate-validation-message')
 
+const validateConfirmationCodeIsCorrect = () => (confirmationCode, { req }) => req.body.confirmationCode === req.session[CONFIRMATION_CODE_SESSION_PROPERTY]
```

Comment:
Could `isCorrect` be removed here, it seems to be implied by `validate`. So function would be called `validateConfirmationCode()`

Reply:
Updated to `validateConfirmationCode()`